### ISSUE TITLE
Creating Custom container and using in another mini app getting node module null in electrodeBridge.js 

### ISSUE THREAD

**Issue by ankit9495** [electrode-io/electrode-native#1027]
While we creating a custom container in our local machine and using the same in a separate mini app , while running in xcode getting error "Native module con not be null" in application.

Node version - v11.4.0
npm version - 6.4.1
xcode version - 9.3.1

**Comment by gmbharath12** [comment]
It means some native modules aren't properly injected into ElectrodeContainer.

Please use the following versions: 
Node `v8.11.3`
npm  `5.6.0`
Xcode `10.1`


How are you adding ElectrodeContainer into a native application ? How's ElectrodeContainer created ?

**Comment by ankit9495** [comment]
Here is the process I followed:
a) Create one miniapp.
b) Create second miniapp and used NavigationAPI of Electrode to communicate between two.
c) Create container by joining two miniapps as

ern create-container --miniapps /Users/Documents/TestUser/TestFolder/ErnMiniSampleMiniApp /Users/Documents/TestUser/TestFolder/SampleMiniApp --platform ios --out /Users/qss/Documents/TestUser/Containers/Samples 

d) Adding this container in our native app by drag and drop.
e) Building project gives me following error:
![screen shot 2018-12-24 at 3 47 00 pm](https://user-images.githubusercontent.com/33280939/50397024-8ab1b600-0793-11e9-8a21-d6825ac8236b.png)



I have updated the versions as you mentioned: 
Node v8.11.3
npm 5.6.0
Xcode 10.0
